pkg/commands/completion.go

```diff
@@ -2,11 +2,46 @@ package commands
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/urfave/cli"
 	"github.com/vapor-ware/synse-cli/pkg/utils"
 )
 
+var bashrc = "$HOME/.bashrc"
+var zshrc = "$HOME/.zshrc"
+
+// bashCompletion is the bash completion function for Synse. This is taken
+// from: https://github.com/urfave/cli/blob/master/autocomplete/bash_autocomplete
+const bashCompletion = `
+_cli_bash_autocomplete() {
+    local cur opts base
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    opts=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
+    COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
+    return 0
+}
+
+complete -F _cli_bash_autocomplete synse
+`
+
+// zshCompletion is the zsh completion function for Synse. This is taken
+// from: https://github.com/urfave/cli/blob/master/autocomplete/zsh_autocomplete
+const zshCompletion = `
+_cli_zsh_autocomplete() {
+
+  local -a opts
+  opts=("${(@f)$(_CLI_ZSH_AUTOCOMPLETE_HACK=1 ${words[@]:0:#words[@]-1} --generate-bash-completion)}")
```

Comment:
Nice env var naming.

Reply:
haha. wasn't me! I just copy and pasted what they had